Menu Icons adapted and adding Translations

henryk86 opened the pull request.

henryk86 commented:
This is my first time working with react... so please be very critical ;)

henryk86 commented:
Is there a chance to get the teddycloud version? I Would love to show the current version somewhere like in the old ui...

SciLor commented:
> Is there a chance to get the teddycloud version? I Would love to show the current version somewhere like in the old ui...
/api/get/internal.version.v_long

To easily get ideas of the API check the network requests. With the way above you can access all settings and internal variable of teddyCloud
https://github.com/toniebox-reverse-engineering/teddycloud/blob/master/src/settings.c

for other endpoints you can check 
https://github.com/toniebox-reverse-engineering/teddycloud/blob/3f301e097afa9a3ae5c243f7835a169e11bccdae/src/server.c#L71

SciLor commented:
> This is my first time working with react... so please be very critical ;)

It would be great to give some more indiciations what you have changed. Picture would be helpful, so I can see the changes without spinning up a npm with your draft.

henryk86 commented:
> > This is my first time working with react... so please be very critical ;)
> 
> It would be great to give some more indiciations what you have changed. Picture would be helpful, so I can see the changes without spinning up a npm with your draft.

Icons:

![image](https://github.com/toniebox-reverse-engineering/teddycloud_web/assets/166761451/0e49bcbd-35fa-4439-8d14-47d916dbd6b8)

![image](https://github.com/toniebox-reverse-engineering/teddycloud_web/assets/166761451/fbd758b6-88e0-427b-b2ba-25e7049ddddf)

Translations:

The other changes are mostly only use translation functionality instead of hardcoded strings.

SciLor commented on public/translations/de.json:
```diff
@@ -21,7 +21,9 @@
       "navigationTitle": "Zertifikate",
-      "title": "Zertifikate"
+      "title": "Zertifikate",
+      "uploadText" : "Klicken oder ziehen Sie die Datei zum Hochladen in diesen Bereich",
```
Sie / du gemischt verwendet. Ich denke alles auf du macht Sinn.

SciLor commented:
Perfect =)

SciLor commented:
Nice one. Could you also add a link to the wiki and telegram? Maybe as a list in the end, including the forum link?

henryk86 commented:
So. added the links and simplified the displayed cards:

![image](https://github.com/toniebox-reverse-engineering/teddycloud_web/assets/166761451/cbe2617f-6973-4614-8c3f-9ed0451903e5)

I am sure, there is a better way than duplicating the tonieCardList and tonieCard files to hide some stuff... feel free to improve

henryk86 commented:
Habe da mal noch ne width beim Menu angegeben... irgendwie wurde egal wie breit das fenster war nur eins der drei Menüeinträge angezeigt...

henryk86 commented:
soooo. done. i stop looking around in the code. no more changes from my side here. please merge ;)
